docs/source/upgrade-guide/index.md

```diff
@@ -48,6 +48,56 @@ You should take the following actions for your Volto 19 projects.
 
 If you can't upgrade immediately, you may continue to run Volto 19 on Node.js 20 at your own risk, but be aware that issues specific to Node.js 20 will not be fixed in the Volto core CI or releases.
 
+(replace-razzle-with-volto-razzle)=
+
+### Replace `razzle` with `@volto/razzle` (fork)
+```{versionchanged} Volto 19.0.0-alpha.10
+```
+
+`@volto/razzle` is a fork of the upstream `razzle` package that contains Volto-specific fixes and patches.
+We recommend using `@volto/razzle` in your Volto 19 projects either when you need the Volto-compatible build behavior or when we provide temporary patches that are not yet merged upstream.
+
+Recommended steps to switch:
+
+1.  Remove the existing `razzle` dependency and install `@volto/razzle` as a development dependency.
+    For example, with `pnpm`:
+
+    ```shell
+    pnpm remove razzle
+    pnpm add -D @volto/razzle
+    ```
+
+2.  Update your {file}`package.json`'s `devDependencies` entry.
+    For example:
+
+    ```diff
+      "devDependencies": {
+    -    "razzle": "4.2.18",
+    +    "@volto/razzle": "x.x.x",
+        ...
+      }
+    ```
+
+3.  In most cases, you don't need to change your scripts—for example `razzle start`, `razzle build`, or `razzle test`—because the fork preserves the original CLI entrypoints.
+    If you have code that imports internal modules from the `razzle` package, for example, `require('razzle/some/path')`), then update those imports to reference `@volto/razzle` instead.
+
+4.  Search your project for any direct or indirect references to `razzle` to ensure nothing was left behind, including imports, requires, and configuration presets or plugins:
```

Comment:
Good to mention, but I think that won't be necessary in 99% of cases. So I'd state that it's only for people that _really_ tapped into it.

Reply:
I think it would have to be changed here:

https://github.com/plone/cookieplone-templates/blob/7b543eafbdcf29a6af71ede34031053aedcb4367/templates/add-ons/frontend/%7B%7B%20cookiecutter.__folder_name%20%7D%7D/packages/%7B%7B%20cookiecutter.frontend_addon_name%20%7D%7D/babel.config.js#L3

So I think it's better to leave it.